packages/next/src/client/next-turbopack.ts

```diff
@@ -0,0 +1,36 @@
+// TODO: Remove use of `any` type.
```

Comment:
can we have top level comment to have quick summary what this file is for? it's ok for us, but if anyone works on next.js and need to amend some for the turbopack might be helpful.